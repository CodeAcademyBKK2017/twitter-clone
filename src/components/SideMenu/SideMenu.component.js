import SideMenuStyle from './SideMenu.style.css';
import React, {Component} from 'react';
import SearchBar from '../SearchBar/SearchBar.component';

class SideMenu extends Component {
	render () {
		return (
			<div className={SideMenuStyle.container}>
				<div className={SideMenuStyle.searchBarContainer}>
					<SearchBar isInSideMenu={true}/>
				</div>
				
				<div className={SideMenuStyle.creditContainer}>
					<b>Made by: Chong</b>
				</div>
			</div>
		);
	}
}

export default SideMenu;
