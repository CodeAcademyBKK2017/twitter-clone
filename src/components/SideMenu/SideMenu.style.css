.container {
	background: #dddddd;
	box-shadow: 0 2px 2px 0 rgba(0,0,0,0.08), 0 0 0 1px rgba(0,0,0,0.04);

	position: absolute;
	left: 0;
	top: 60px;
	width: 80vw;
	height: calc(100vh - 100%);

	display: flex;
	flex-direction: column;
	align-items: stretch;
}

.hidden {
	display: none;
}

.searchBarContainer {
	padding: 10px;
}

.menuContainer {
	flex: 1;
}

.creditContainer {
	padding: 10px;

	border-top: solid 1px rgba(128, 128, 128, 0.5);
	color: gray;
	text-align: center;
}

@media all and (max-width: 768px) {
	
}
