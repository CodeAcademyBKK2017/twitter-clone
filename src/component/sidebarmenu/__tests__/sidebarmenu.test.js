import SideMenu from '../sidebarmenu.component';
import React from 'react';
import renderer from 'react-test-renderer';

describe('SideMenu component', () => {
  it('SideMenu: renders correctly', () => {
    const tree = renderer.create(<SideMenu />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});