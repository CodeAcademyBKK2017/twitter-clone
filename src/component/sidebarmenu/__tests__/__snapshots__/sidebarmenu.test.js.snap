// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideMenu component SideMenu: renders correctly 1`] = `
<div
  className=""
>
  <div
    className=""
  >
    <div />
  </div>
  <div
    className=""
  >
    Made By Kingdark
  </div>
</div>
`;
